client/client-multi/client-objective-c/Kaa/KaaClientFactory.h

```diff
@@ -0,0 +1,44 @@
+/**
+ *  Copyright 2014-2016 CyberVision, Inc.
```

Comment:
What deleted file? This header (and its implementation file) was called Kaa. But framework requires to have special header file with the same name as framework, in this case Kaa.h. So old Kaa.h (and Kaa.m) was renamed to KaaClientFactory.h, and header imports from it were moved to new Kaa.h file.